syntax = "proto2";
package mmo.proto;

message TalentEntry {
	required uint32 id = 1;
	required uint32 tab = 2;
	required uint32 row = 3;
	required uint32 column = 4;
	repeated uint32 ranks = 5;	// Spell-IDs
	optional uint32 dependson = 6;
	optional uint32 dependsonrank = 7;
	optional uint32 dependsonspell = 8;
}

message Talents {
	repeated TalentEntry entry = 1;
}
